yt_dlp/extractor/tver.py

```diff
@@ -3,75 +3,40 @@
-        'only_matching': True,
-    }, {
-        # subtitle = ' '
-        'url': 'https://tver.jp/corner/f0068870',
-        'only_matching': True,
-    }, {
-        'url': 'https://tver.jp/lp/f0009694',
-        'only_matching': True,
-    }, {
-        'url': 'https://tver.jp/lp/c0000239',
-        'only_matching': True,
-    }, {
-        'url': 'https://tver.jp/tokyo2020/video/6264525510001',
+        'url': 'https://tver.jp/episodes/ephss8yveb',
         'only_matching': True,
     }]
-    _TOKEN = None
     BRIGHTCOVE_URL_TEMPLATE = 'http://players.brightcove.net/%s/default_default/index.html?videoId=%s'
 
-    def _real_initialize(self):
-        self._TOKEN = self._download_json(
-            'https://tver.jp/api/access_token.php', None)['token']
-
     def _real_extract(self, url):
-        path, video_id = self._match_valid_url(url).groups()
-        if path == 'lp':
-            webpage = self._download_webpage(url, video_id)
-            redirect_path = self._search_regex(r'to_href="([^"]+)', webpage, 'redirect path')
-            path, video_id = self._match_valid_url(f'https://tver.jp{redirect_path}').groups()
-        api_response = self._download_json(f'https://api.tver.jp/v4/{path}/{video_id}', video_id, query={'token': self._TOKEN})
-        p_id = traverse_obj(api_response, ('main', 'publisher_id'))
-        if not p_id:
-            error_msg, expected = traverse_obj(api_response, ('episode', 0, 'textbar', 0, ('text', 'longer')), get_all=False), True
-            if not error_msg:
-                error_msg, expected = 'Failed to extract publisher ID', False
-            raise ExtractorError(error_msg, expected=expected)
-        service = remove_start(traverse_obj(api_response, ('main', 'service')), 'ts_')
-
-        r_id = traverse_obj(api_response, ('main', 'reference_id'))
-        if service not in ('tx', 'russia2018', 'sebare2018live', 'gorin'):
-            r_id = 'ref:' + r_id
+        video_id = self._match_id(url)
+        video_info = self._download_json(
+            f'https://statics.tver.jp/content/episode/{video_id}.json', video_id,
+            query={'v': '5'}, headers={
+                'Origin': 'https://tver.jp',
+                'Referer': 'https://tver.jp/',
+            })
+        p_id = traverse_obj(video_info, ('video', 'accountID'))
+        r_id = 'ref:' + traverse_obj(video_info, ('video', 'videoRefID'))
         bc_url = smuggle_url(
             self.BRIGHTCOVE_URL_TEMPLATE % (p_id, r_id),
             {'geo_countries': ['JP']})
 
         return {
             '_type': 'url_transparent',
-            'description': traverse_obj(api_response, ('main', 'note', 0, 'text'), expected_type=compat_str),
-            'episode_number': int_or_none(traverse_obj(api_response, ('main', 'ext', 'episode_number'), expected_type=compat_str)),
+            'title': str_or_none(video_info.get('title')),
+            'description': str_or_none(video_info.get('description')),
             'url': bc_url,
```

Comment:
since these are mandatory fields, traverse_obj is not needed
```suggestion        
        p_id, r_id = video_info['video']['accountID'], video_info['video']['videoRefID']

        return {
            '_type': 'url_transparent',
            'url': smuggle_url(self.BRIGHTCOVE_URL_TEMPLATE % (p_id, f'ref:{r_id}'), {'geo_countries': ['JP']}),
            'title': str_or_none(video_info.get('title')),
            'description': str_or_none(video_info.get('description')),
```

Reply:
but I won't for r_id, see another review below

Reply:
yeah, this is just some refactoring. ignore this till the actual extraction is fully figured out